packages/devtools-modules/src/menu/index.js

```diff
@@ -4,6 +4,10 @@
 
 const EventEmitter = require("../utils/event-emitter");
 
+function inToolbox() {
+  return window.parent.document.documentURI == "about:devtools-toolbox"
```

Comment:
Not sure if this check is future-proof, but if other devtoolers are ok with it, then it should be fine :)